pkg/workflow/providers/multicluster/deploy.go

```diff
@@ -186,8 +339,25 @@ func applyComponents(ctx context.Context, apply oamProvider.ComponentApply, heal
 		taskHealthyMap[tasks[i].key()] = res.healthy
 	}
 
+	reapplyTask := getReapplyTasks(tasks, taskHealthyMap)
+
+	var outputResult []*applyTaskResult
+	if len(reapplyTask) > 0 {
```

Comment:
reapplyTasks are done tasks but need to reapply and get outputs?

Reply:
yes, or I guess we have to do changes to make `healthCheck` to return the manifests. This change
1. will change a lot places except deploy step.
2. make the function less cohesive